features/ubuntu_pro.feature

```diff
@@ -186,6 +186,16 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
             ros           +no  +(-|—) +Security Updates for the Robot Operating System
             ros-updates   +no  +(-|—) +All Updates for the Robot Operating System
             """
+        When I run `systemctl start ua-auto-attach.service` with sudo
```

Comment:
I think we also add a seconf `ua auto-attach` step here, just to show the command failing in a PRO machine

Reply:
oki, can do